package io.github.clamentos.cachecruncher.error.exceptions;

///
public final class ValidationException extends CacheCruncherException {

    ///
    public ValidationException(Throwable cause) {

        super(cause);
    }

    ///
}
